tracer/build/_build/Build.GitHub.cs

```diff
@@ -535,7 +535,7 @@ LabbelerConfiguration GetLabellerConfiguration()
                 var fixIssues = new[] { "type:bug", "type:regression", "type:cleanup" };
                 var areaLabelToComponentMap = new Dictionary<string, string>() {
                     { "area:tracer", tracer },
-                    { "area:ci-app", ciApp },
+                    { "area:ci-visibility", ciVisibility },
```

Comment:
you should change the [labeller config](https://github.com/DataDog/dd-trace-dotnet/blob/master/.github/labeller.yml#L2) as well

Reply:
oh, thanks!, done in [47385b2](https://github.com/DataDog/dd-trace-dotnet/pull/3046/commits/47385b26ad61ee06fa901c36b622f48cfbe10c8b)